tt_eager/tt_dnn/op_library/moreh_layernorm_backward/moreh_layernorm_backward_op.cpp

```diff
@@ -120,30 +133,34 @@ void MorehLayerNormBackwardGammaBetaGrad::validate(
 
 std::vector<Shape> MorehLayerNormBackwardGammaBetaGrad::compute_output_shapes(
     const std::vector<Tensor>& input_tensors) const {
-    // Inplace
+    // Not Implemented
```

Comment:
use tt_fatal here too

Reply:
```C++
T_FATAL(false, "The compute_output_shapes function in MorehLayerNormBackwardGammaBetaGrad is not implemented.");
```
I have made the changes as shown in the code above.